doc/en/cache.rst

```diff
@@ -67,21 +67,21 @@ If you run this for the first time you will see two failures::
     E          Failed: bad luck
     
     test_50.py:6: Failed
-    2 failed, 48 passed in 0.04 seconds
+    2 failed, 48 passed in 0.12 seconds
 
 If you then run it with ``--lf``::
 
     $ py.test --lf
-    =========================== test session starts ============================
-    platform linux2 -- Python 2.7.6, pytest-2.7.3.dev428+ng79d22bf.d20150916, py-1.4.30, pluggy-0.3.0
+    ======= test session starts ========
+    platform linux2 -- Python 2.7.10, pytest-2.8.1.dev1, py-1.4.30, pluggy-0.3.1
```

Comment:
Unrelated to the PR, but I think we should promote py3 and generate the docs using py35... what you guys think?